src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/DefaultTagHelperServiceTestBase.cs

```diff
@@ -6,6 +6,11 @@
 using Microsoft.AspNetCore.Razor.Language.Components;
 using Microsoft.AspNetCore.Razor.Test.Common;
 using Microsoft.VisualStudio.Editor.Razor;
+using Moq;
```

Comment:
nit: remove and sort usings

Reply:
This is removed and sorted. Is there a specific rule you feel it's violating, possibly my remove and sort settings are off.

If you're just thinking that's way too many adds for the small amount of changes the culprit is all the classes I new-upped for the Language server params.

Reply:
lol my brain totally saw the line below 

`using DefaultRazorTagHelperCompletionService = Microsoft.VisualStudio.Editor.Razor.DefaultTagHelperCompletionService;`

And thought it wasn't sorted because it didn't register that it was a using alias. Ignore this comment!